mne/preprocessing/ica.py

```diff
@@ -214,6 +215,9 @@ class ICA(ContainsMixin):
         The number of components used for PCA dimensionality reduction.
     verbose : bool, str, int, or None
         See above.
+    pre_whitener_ : ndarray
```

Comment:
you really don't want to add the shapes here?

Reply:
I do want to add the shapes eventually, I was just saying that this might be better implemented in #4856 once we've sorted out all open issues there. However, if you insist I can also add them here. Let me know.